sdk/identity/azure-identity/TOKEN_CACHING.md

```diff
@@ -18,6 +18,10 @@ The in-memory token cache provided by the Azure Identity library can be used by
 
 **Note:** When Azure Identity library credentials are used with Azure service libraries (for example, Azure Blob Storage), the in-memory token caching is active in the `HttpPipeline` layer as well. All `TokenCredential` implementations are supported there, including custom implementations external to the Azure Identity library.
 
+#### Caching cannot be disabled
```

Comment:
Should this be `Disabling Token Caching` to be more consistent with other headers